DocumentiPubblici/AnalisiDeiRequisiti/Sezioni/CasiUso/1AccessoAllaPiattaforma.tex

```diff
@@ -26,29 +26,127 @@
+L'utente non autenticato che dispone di credenziali venditore può accedere nella piattaforma usando l'email e la password.
 \begin{itemize}
     \item \textbf{Attori Primari:} Utente non autenticato.
-    \item \textbf{Precondizione:} L'utente è già presente nella piattaforma e si trova nella pagina di login.
-    \item \textbf{Postcondizione:} L'utente è autenticato.
-    \item \textbf{Scenario Principale:} L'utente accede alla piattaforma compilando tutti i campi del modulo di login nel seguente modo:
+    \item \textbf{Precondizione:} L'utente dispone di credenziali venditore e si trova nella pagina di login.
+    \item \textbf{Postcondizione:} L'utente è autenticato come venditore e si trova sulla dashboard.
+    \item \textbf{Scenario Principale:} L'utente accede alla piattaforma con delle credenziali in questo modo:
     \begin{itemize}
-        \item (UC23.1) - Inserimento indirizzo email.
-        \item (UC23.2) - Inserimento password.
+        \item (\actualUC.1) - Inserimento indirizzo.
+        \item (\actualUC.2) - Inserimento password.
+        \item Richiede l'autenticazione.
+        \item L'utente è autenticato come venditore.
+        \item L'utente viene portato alla dashboard.
+    \end{itemize}
+	\item \textbf{Scenario Alternativo:} l'utente compila i campi dati con delle credenziali 
+	\begin{itemize}
+		\item (\actualUC.1) - Inserimento indirizzo email.
+		\item (\actualUC.2) - Inserimento password.
+        \item Richiede l'autenticazione.
     \end{itemize}
+	Ma le credenziali fornite non appartengono alla piattaforma, quindi:
+	\begin{itemize}
+		\item viene visualizzato un messaggio di errore apposito (UC estensione)
+		\item l'utente rimane non autenticato e alla pagina di login.
+	\end{itemize}
     \item \textbf{Estensioni:}
     \begin{itemize}
-        \item (UC26) - Visualizzazione messaggio di errore in caso di email o password errata.
+        \item (UC) - Visualizzazione messaggio di errore in caso di email o password errata.
     \end{itemize}
+    \item \textbf{Inclusioni:}
+    \begin{itemize}
+    	\item \actualUC.1 - Inserimento indirizzo email.
+    	\item \actualUC.2 - Inserimento password. 
+    \end{itemize}
+\end{itemize}
+
+\resetSubUC
+\subUC{Inserimento indirizzo email}
+\begin{itemize}
+	\item \textbf{Attori Primari:} Utente non autenticato.
+	\item \textbf{Precondizione:} L'utente non è autenticato
+	\item \textbf{Postcondizione:} L'utente ha inserito il proprio indirizzo email.
+	\item \textbf{Scenario Principale:} inserimento email per l'autenticazione.
+\end{itemize}
+
+\subUC{Inserimento password}
+\begin{itemize}
+	\item \textbf{Attori Primari:} Utente non autenticato.
+	\item \textbf{Precondizione:} L'utente non è autenticato
+	\item \textbf{Postcondizione:} L'utente ha inserito la propria password.
+	\item \textbf{Scenario Principale:} inserimento password per l'autenticazione.
+\end{itemize}
```

Comment:
Servono davvero? Perchè cardin aveva detto di lasciar stare gli UC che non danno alcuna informazione in più

Reply:
aggiungo il messaggio di campo obligatorio per entrambi, mi sembra sensato visto che sono inclusioni

Reply:
A proposito, perchè sono inclusioni? cardin ci aveva detto parole quando li avevamo messi